docker/docker/Dockerfile.linux.amd64

```diff
@@ -1,4 +1,4 @@
-FROM docker:18.06-dind
+FROM amd64/docker:18.06-dind
```

Comment:
I think you can keep it at `docker:18.06-dind`, that's what we are doing for all the other plugins as well. We just map the other architectures to the separate namespaces.

Reply:
OK. revered.